Scripts/RemediationScripts/Remediate-EnableAADForKubernetesService.ps1

```diff
@@ -743,13 +743,13 @@ class SetAADProfile
             }
             
 
-            # API to set AAD Profile config to Kubernetes cluster
+            # API to set Entra Id Profile config to Kubernetes cluster
             $response = Invoke-WebRequest -Method $method -Uri $armUri -Headers $headers -Body $jsonString -UseBasicParsing
             $content = ConvertFrom-Json $response.Content
         }
         catch
         {
-            Write-Host "Error occurred while setting AAD profile to Kubernetes Cluster. ErrorMessage [$($_)]" -ForegroundColor $([Constants]::MessageType.Error)
+            Write-Host "Error occurred while setting Entra Id profile to Kubernetes Cluster. ErrorMessage [$($_)]" -ForegroundColor $([Constants]::MessageType.Error)
```

Comment:
Again, need a VSCode root repository search on this.
Are you sure you're covering all variants of AAD?
1. AAD
2. Azure Active Directory